inscriptions/ISic003866.xml

```diff
@@ -77,26 +74,37 @@
                         <objectDesc>
                             <supportDesc>
                                 <support>
-                                    <dimensions><!--Default values and requires editing-->
-                                        <height unit="cm"/>
-                                        <width unit="cm"/>
-                                        <depth unit="cm"/>
+                                    <p>Carefully squared and polished stele, with traces on the base for recessing in a plinth, slightly chipped on the edges. Plinth made separately in stone from Monte Rosa, with irregular edges and surfaces, incomplete on the left side of the front face and chipped.</p>
+                                    <material ana="#material.stone.volcanic" ref="http://www.eagle-network.eu/voc/material/lod/28.html">volcanic (pietra di Fuardo)</material>
+                                    <objectType ana="#object.stele" ref="http://www.eagle-network.eu/voc/objtyp/lod/250">stele</objectType>
+                                    <dimensions>
+                                        <height unit="cm">68.5</height>
+                                        <width unit="cm">45.5</width>
+                                        <depth unit="cm">11.5</depth>
+                                    </dimensions>
+                                    
+                                    <p>Plinth:</p>
```

Comment:
this would seem to me to confuse things slightly, since it's not a formal model we've adopted and would confuse any attempt to use the dimensions for machine-based analysis?

Reply:
Yes, I left it like that as an example for further discussion. I might end up just having the plinth description in a separate paragraph.